.github/workflows/ompi_mpi4py.yaml

```diff
@@ -127,11 +127,9 @@ jobs:
 
   run_spawn:
```

Comment:
Should the spawn tests just be merged up into the main chunk of tests?  The only reason to run this chunk separately was so that it could fail, and we wouldn't care.  But if the spawn tests are now expected to pass, we might as well run them up in the main tests, and not have a separate section for the spawn tests.

Reply:
I don't have a strong opinion, though I prefer to keep the test separate for its record in the past.

@hppritcha What do you think?

Reply:
Isn't it "cheaper" to do a single github action run?  I vote for merging.